cmd/usecasevalidator/main.go

```diff
@@ -75,6 +75,7 @@ const anytypeProfileFilename = addr.AnytypeProfileId + ".pb"
 var (
 	errIncorrectFileFound = fmt.Errorf("incorrect protobuf file was found")
 	errValidationFailed   = fmt.Errorf("validation failed")
+	errSkipObject         = fmt.Errorf("validation failed, but object can be skipped")
```

Comment:
Choose another error message please. Something like "object is invalid, skip it".
It's just misleading a bit imho. Why don't we failed validation if we have error about failed validation 😅

Reply:
Yeah, thank you, changed the error message